watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/amo_1595_photothermal_membrane_candoP/amo_1595.py

```diff
@@ -281,16 +295,30 @@ def add_costing(m):
             )
             * m.fs.costing.utilization_factor
         ),
-        doc="Levelized Cost of Water when accounting for salable BCP",
+        doc="Levelized Cost of Water including revenue from BCP and N20",
     )
 
     m.fs.costing.LCOT = Expression(
+        expr=(
+            m.fs.costing.total_capital_cost * m.fs.costing.capital_recovery_factor
+            + m.fs.costing.total_operating_cost
+        )
+        / (
+            pyunits.convert(
+                m.fs.feed.properties[0].flow_vol,
+                to_units=pyunits.m**3 / m.fs.costing.base_period,
+            )
+            * m.fs.costing.utilization_factor
+        ),
+        doc="Levelized Cost of Treatment with respect to influent flowrate",
+    )
+
+    m.fs.costing.LCOT_with_revenue = Expression(
         expr=(
             m.fs.costing.total_capital_cost * m.fs.costing.capital_recovery_factor
             + m.fs.costing.total_operating_cost
             - m.fs.costing.value_bcp_recovery
             - m.fs.costing.value_N2O_byproduct
-            - m.fs.costing.value_water_byproduct
```

Comment:
I think that in this particular project, water recovery is more of a bonus, while the focus is placed on resource recovery. For that reason, I would lean towards keeping the value of water. Worst case scenario, the default sale price for water could be a very small number representative of 0, thus assuming no value for water but allowing the user to do a sensitivity analysis on that sale price.

Reply:
Addressed in latest commit.